scripts/irods/test/test_iticket.py

```diff
@@ -766,3 +767,86 @@ def test_write_tickets_for_collections_allow_creation_of_data_objects__issue_591
                 self.admin.run_icommand(['ichmod', '-M', 'own', self.admin.username, fname])
 
             self.admin.run_icommand(['iticket', 'delete', ticket_name])
+
+    def test_tickets_update_catalog_modify_time_when_modified__issue_7125(self):
+
+        def get_modification_and_creation_time():
+            out, err, ec = self.admin.run_icommand(['iquest', '%s...%s', "select TICKET_MODIFY_TIME, TICKET_CREATE_TIME where TICKET_STRING = '{}'".format(ticket_string)])
+            self.assertEqual(ec, 0)
+            self.assertEqual(len(err), 0)
+            modification_time = out.strip().split('...')[0]
+            creation_time = out.strip().split('...')[1]
+
+            time.sleep(1) # necessary because epoch time is in seconds
```

Comment:
Consider pulling this out into the main test logic or rename the utility function so it's clear what's happening.

At the moment, this utility function is serving two jobs. (1) Get the mtime/ctime. (2) Sleep so there's a gap between mtime/ctime checks.

This hides info from the next reader of this test.